libs/server/user/infrastructure/src/lib/mappers/map-github-user.ts

```diff
@@ -0,0 +1,11 @@
+import { plainToClass } from '@marcj/marshal';
+import { User } from '@pimp-my-pr/server/user/core/domain';
+import { GithubUser } from '../repositories/github/github-user.entity';
+
+export function mapGithubUser(githubUser: GithubUser): User {
+  return plainToClass(User, {
+    avatarUrl: githubUser.avatar_url,
+    id: githubUser.id,
```

Comment:
`id` property in GithubUser interface is a number but in User class it's declared as string. Shouldn't numeric `id` be converted to string in this place?

Reply:
It's not a problem. Marshal mapper will handle that. The user will have Id in the String type.